src/main/java/com/github/vkorobkov/jfixtures/config/structure/tables/Tables.java

```diff
@@ -42,6 +42,13 @@ public CleanMethod getCleanMethod() {
         return instructions;
     }
 
+    public List<String> getAfterInserts() {
```

Comment:
This is common code with `getBeforeInserts`.

Maybe it makes sense to extract a private method?
```java
List<String> readArrayRecursively(String... sections)
```